pad.go

```diff
@@ -103,3 +103,17 @@ func (pad *PAD) GetSTR(epoch uint64) *SignedTreeRoot {
 	}
 	return pad.snapshots[epoch]
 }
+
+func (pad *PAD) TB(key string, value []byte) *TemporaryBinding {
+	index := computePrivateIndex(key)
+	tb := pad.currentSTR.sig
+	tb = append(tb, index...)
+	tb = append(tb, value...)
+	sig := crypto.Sign(pad.key, tb)
+
```

Comment:
Should we also call `pad.Set(key, value)` in here?  If we're issuing a TB, it's kind of implying that, no?